include/ipv6.h

```diff
@@ -68,6 +73,11 @@ struct inet6_protocol {
     unsigned int    flags;
 };
 
+int ip6_skip_exthdr(const struct rte_mbuf *imbuf, int start,
+                    __u8 *nexthdrp);
```

Comment:
function int ip6_skip_exthdr(const struct rte_mbuf *mbuf, int start, uint8_t *nexthdrp); should be deleted

Reply:
ok